stories/layout/stack.js

```diff
@@ -44,7 +44,7 @@ export default function() {
   const sections = number('Sections', 2, { range: true, min: 2, max: 10 })
 
   return (
-    <Layout type="parent" className={style.page}>
+    <Layout type="parent" className={style.page} style={{ maxHeight: '100%' }}>
```

Comment:
I know we don't care about preventing inline styles for devtools like storybook, but is this a pattern we want to allow?